extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -432,6 +443,8 @@ public void actionPerformed (ActionEvent e) {
 		JLabel extensionsLabel = new JLabel("Extensions");
 		List<JPanel> extensionsPanels = new ArrayList<JPanel>();
 
+        SetupButton showMoreExtensionsButton = new SetupButton("Show More");
```

Comment:
That would be a huge button I image you guys want it to keep less noticeable. Maybe "Third Party" ?